projects/kit/components/data-list-wrapper/data-list-wrapper.template.html

```diff
@@ -2,8 +2,9 @@
     *ngIf="items; else loading"
     [emptyContent]="emptyContent"
 >
+    <!-- TODO: need fix type of item - T | T[] -->
     <button
-        *ngFor="let item of items"
+        *ngFor="let item of $any(items)"
```

Comment:
It is `T` for `data-list-wrapper` and `T[]` for `data-list-group-wrapper`, two separate components.

Reply:
fixed